skxray/core/fitting/xrf_model.py

```diff
@@ -1143,10 +1110,8 @@ def linear_spectrum_fitting(x, y, params,
             e.g., ['Na_K', Mg_K', 'Pt_M'] refers to the
```

Comment:
I know that this is not part of this PR, but can you correct this docstring?

`option` -> `optional`

Can you also describe what the default behavior is going to be because this is an optional parameter?